com.basistech.m2e.code.quality.shared/src/main/java/com/basistech/m2e/code/quality/shared/AbstractMavenPluginConfigurationTranslator.java

```diff
@@ -140,6 +143,22 @@ protected URL resolveLocation(String resc) {
 		return this.resourceResolver.resolveLocation(resc);
 	}
 
+	protected void copyIfChanged(InputStream input, Path output) throws IOException {
+		byte[] fileContent = IOUtil.toByteArray(input);
+		ByteArrayInputStream bufferedInput = new ByteArrayInputStream(fileContent);
```

Comment:
This bufferedInput never gets closed, but because it's just backed by a byte[] and not a file, that's probably OK? Perhaps just for consistency still better to have it in try (ByteArrayInputStream bufferedInput = new ByteArrayInputStream(fileContent)) anyway?

Reply:
yes, the `close()` is a NOOP. But I can put it in a try-with-resources block